1 HW_Standard/HW_Breloom.yaml

```diff
@@ -0,0 +1,22 @@
+species: Breloom
```

Comment:
Far too bulky to carry fissure, cut out the bulk investment at minimum.

Reply:
Trying to parse this out. Should I just zero out the IVs, since in normal play most people would just use the other moves over fissure?

Reply:
While luck based, fissue won't always be ignored. It basically always 3hko's it target on average and if you are playing the odds that's powerful, especially on bulky things and against bulky things. Riven the recent clarification that holiday sets are weaker than the norm I would say remove the IVs too.